src/views/course-edit-tools/course-feedback.vue

```diff
@@ -58,43 +61,42 @@ export default {
 
   computed: {
     ...mapActions([
-      'fetchEnrollment'
+      'fetchFeedbackDownload'
     ]),
     ...mapGetters([
       'enrollmentFeedback',
-      'courseId'
+      'course'
     ])
   },
 
   methods: {
     /**
      * Function printPDF: prints feedback in a PDF tp download
      * Author: nv
-     * Last Updated: September 14, 2022
+     * Last Updated: September 28, 2022 by nv
      */
     async printPDF () {
       if (typeof this.enrollmentFeedback !== 'undefined') {
         /* eslint-disable */
         const doc = new jsPDF('p','pt')
         /* eslint-enable */
-        const id = this.courseId
-        let name
         let feedback = []
+        const name = this.course.name
         const x = 30
         let y = 30
         const lineheight = doc.getTextDimensions('Sample Text').h + 5
         let blockheight = 0
+        const promise = this.$store.dispatch('fetchFeedbackDownload', name)
 
-        await http.get(`courses/getCourseName?courseId=${id}`)
-          .then(({ data }) => {
-            console.log(data.courseName)
-            name = data.courseName
-          })
-        await http.get(`enrollments/getAllByCourseId?courseId=${id}`)
-          .then(({ data }) => {
-            console.log(data.subs[0].feedback)
-            feedback = data.subs[0].feedback
-          })
+        for (let i = 0; i < promise.subs.length; i++) {
+          feedback = promise.subs[i].feedback
+        }
```

Comment:
you could move this to the store action, then the name would make sense

prefer `for in` or `for of` notation for readability